webapp/pages/terms-and-conditions.vue

```diff
@@ -14,10 +15,16 @@
 
 <script>
 export default {
+  layout: 'blank',
   head() {
     return {
       title: this.$t('site.termsAc'),
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-07-19T13:35:48Z" title="Friday, July 19th 2019, 3:35:48 pm +02:00">Jul 19, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -1,14 +1,72 @@
 <template>
   <div>
     <ds-space>
-      <ds-heading tag="h2">{{ $t('site.termsAc') }}</ds-heading>
+      <ds-heading tag="h2">{{ $t('site.termsAc') }} ( vom: {{ latestDate_TaC }})</ds-heading>
     </ds-space>
+    <div v-if="confirmed === true">
+      <ds-section class="hasConfirm" primary>
+        <ds-flex>
+          <ds-flex-item>
+            <ds-logo inverse></ds-logo>
+          </ds-flex-item>
+          <ds-flex-item width="4">
+            <ds-heading>you have confirmed</ds-heading>
+            <ds-text>
```


```suggestion
            <ds-text v-if="confirmed">
```

**This** is where I would jump in with an if-clause. Because you have to replace the whole text.